{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/SnowplowTracker\/documentation\/SnowplowTracker\/MediaErrorEvent\/errorDescription(_:)"},"kind":"symbol","schemaVersion":{"major":0,"minor":3,"patch":0},"primaryContentSections":[{"declarations":[{"tokens":[{"text":"@objc","kind":"attribute"},{"text":" ","kind":"text"},{"text":"func","kind":"keyword"},{"kind":"text","text":" "},{"text":"errorDescription","kind":"identifier"},{"text":"(","kind":"text"},{"text":"_","kind":"externalParam"},{"text":" ","kind":"text"},{"kind":"internalParam","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?) -> "},{"text":"Self","kind":"typeIdentifier"}],"languages":["swift"],"platforms":["macOS"]}],"kind":"declarations"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/snowplowtracker\/mediaerrorevent\/errordescription(_:)"]}],"sections":[],"metadata":{"modules":[{"name":"SnowplowTracker"}],"title":"errorDescription(_:)","externalID":"c:@M@SnowplowTracker@objc(cs)SPMediaErrorEvent(im)errorDescription:","fragments":[{"kind":"keyword","text":"func"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"?) -> ","kind":"text"},{"text":"Self","kind":"typeIdentifier"}],"symbolKind":"method","roleHeading":"Instance Method","role":"symbol"},"hierarchy":{"paths":[["doc:\/\/SnowplowTracker\/documentation\/SnowplowTracker","doc:\/\/SnowplowTracker\/documentation\/SnowplowTracker\/MediaErrorEvent"]]},"abstract":[{"type":"text","text":"Longer description for the error that occurred in the playback."}],"references":{"doc://SnowplowTracker/documentation/SnowplowTracker/MediaErrorEvent":{"fragments":[{"text":"class","kind":"keyword"},{"kind":"text","text":" "},{"text":"MediaErrorEvent","kind":"identifier"}],"url":"\/documentation\/snowplowtracker\/mediaerrorevent","type":"topic","role":"symbol","title":"MediaErrorEvent","kind":"symbol","abstract":[{"text":"Media player event tracked when the resource could not be loaded due to an error.","type":"text"}],"identifier":"doc:\/\/SnowplowTracker\/documentation\/SnowplowTracker\/MediaErrorEvent","navigatorTitle":[{"text":"MediaErrorEvent","kind":"identifier"}]},"doc://SnowplowTracker/documentation/SnowplowTracker/MediaErrorEvent/errorDescription(_:)":{"title":"errorDescription(_:)","role":"symbol","kind":"symbol","fragments":[{"text":"func","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":"("},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"?) -> ","kind":"text"},{"text":"Self","kind":"typeIdentifier"}],"type":"topic","identifier":"doc:\/\/SnowplowTracker\/documentation\/SnowplowTracker\/MediaErrorEvent\/errorDescription(_:)","url":"\/documentation\/snowplowtracker\/mediaerrorevent\/errordescription(_:)","abstract":[{"type":"text","text":"Longer description for the error that occurred in the playback."}]},"doc://SnowplowTracker/documentation/SnowplowTracker":{"title":"SnowplowTracker","role":"collection","kind":"symbol","type":"topic","identifier":"doc:\/\/SnowplowTracker\/documentation\/SnowplowTracker","url":"\/documentation\/snowplowtracker","abstract":[]}}}